Packs/XMCyber/Integrations/XMCyberIntegration/XMCyberIntegration.py

```diff
@@ -20,89 +20,107 @@
+XM_CYBER_INCIDENT_TYPE_ASSET = "XM Cyber Critical Asset"
+XM_CYBER_INCIDENT_TYPE_CHOKE_POINT = "XM Cyber Choke Point"
 TOP_ENTITIES = 3
 PAGE_SIZE = 50
 MAX_PAGES = 10
 SENSOR_TYPE = "Sensor"
 
-''' CLIENT CLASS '''
+XmEventType = Dict[str, Any]
+""" CLIENT CLASS """
 
 
 class Client(BaseClient):
     """Client class to interact with XM Cyber API"""
 
-    def get(self, url_suffix, params=None):
-        return self._http_request(
-            method='GET',
-            url_suffix=url_suffix,
-            params=params
-        )
+    def get(self, url_suffix: str, params: Optional[Dict[str, Any]] = None):
+        return self._http_request(method="GET", url_suffix=url_suffix, params=params)
 
-    def post(self, url_suffix, params=None):
+    def post(self, url_suffix: str, params: Optional[Dict[str, Any]]):
         return self._http_request(
-            method='POST',
-            url_suffix=url_suffix,
-            data=json.dumps(params)
+            method="POST", url_suffix=url_suffix, data=json.dumps(params)
         )
 
-    def _paginated(self, method, url_suffix, params, page_size, max_pages, log):
+    def _paginated(
+        self,
+        method: str,
+        url_suffix: str,
+        params: Dict[str, Any],
+        page_size: int,
+        max_pages: int,
+        log: bool,
+    ):
         page, total_pages = 1, 1
         data = []
         params["pageSize"] = page_size
         while page <= total_pages and page <= max_pages:
             if log:
-                writeLog(f'in {url_suffix} while loop {page} {total_pages}')
+                writeLog(f"in {url_suffix} while loop {page} {total_pages}")
 
             params["page"] = page
             res = None
-            if method == 'GET':
+            if method == "GET":
                 res = self.get(url_suffix, params)
             else:
                 res = self.post(url_suffix, params)
 
-            data.extend(res['data'])
-            total_pages = res['paging']['totalPages']
+            data.extend(res["data"])
+            total_pages = res["paging"]["totalPages"]
             page += 1
 
         return data
 
-    def get_paginated(self, url_suffix, params={}, page_size=PAGE_SIZE, max_pages=MAX_PAGES, log=True):
+    def get_paginated(
+        self,
+        url_suffix: str,
+        params: Optional[Dict[str, Any]] = None,
+        page_size: int = PAGE_SIZE,
+        max_pages: int = MAX_PAGES,
+        log: bool = True,
+    ):
+        if params is None:
+            params = {}
         return self._paginated("GET", url_suffix, params, page_size, max_pages, log)
 
-    def post_paginated(self, url_suffix, params={}, page_size=PAGE_SIZE, max_pages=MAX_PAGES, log=True):
+    def post_paginated(
+        self,
+        url_suffix: str,
+        params: Optional[Dict[str, Any]] = None,
+        page_size: int = PAGE_SIZE,
+        max_pages: int = MAX_PAGES,
+        log: bool = True,
+    ):
+        if params is None:
+            params = {}
```

Comment:
Mutable default args in python are pretty dangerous. In this particular case we are not changing the `params` value, so it will be ok, but in general I prefer not to do that...